source/core/parameters_lagrangian.cc

```diff
@@ -656,15 +656,15 @@ namespace Parameters
           "particle particle contact force method",
           "hertz_mindlin_limit_overlap",
           Patterns::Selection(
-            "linear|hertz_mindlin_limit_force|hertz_mindlin_limit_overlap|hertz|hertz_JKR"),
+            "linear|hertz_mindlin_limit_force|hertz_mindlin_limit_overlap|hertz|hertz_JKR|DMT"),
```

Comment:
It is better to use parameters as lowercase.

Reply:
Also I think this will change with my suggestion above. I mean: cohesion model as boolean and the model options as jkr and dmt

Reply:
JKR is written JKR. 

It's coherent to use DMT.